fiasko_bro/code_validator.py

```diff
@@ -146,6 +146,11 @@ def run_validator_group(self, group, add_warnings=False, *args, **kwargs):
             return errors
         return errors
 
+    def get_syntax_errors_in_repo(self, repo):
```

Comment:
Now you can simply get rid of the method. Take a closer look at the code in `validate`: doesn't it do the same thing?